src/components/product.js

```diff
@@ -593,7 +593,7 @@ export default class Product extends Component {
       }
 
       this.props.client.checkout.create().then((checkout) => {
-        const lineItem = {variantId: this.selectedVariant.id, quantity: 1};
+        const lineItem = {variantId: this.selectedVariant.id, quantity: this.selectedQuantity};
```

Comment:
is `this.selectedQuantity` defaulted anywhere?

Reply:
yup, it's done in the product's constructor: https://github.com/Shopify/buy-button-js/blob/c5b9c87f64f87cdcbcc7fa2d12c5f38cebb5afe6/src/components/product.js#L79

Reply:
👍